scripts/trainer.py

```diff
@@ -36,11 +28,8 @@ def download_db(self, db_type):
         path = f"data/{db_type}.json"
         formatted_base_url = BASE_URL.format(db_type, f"v{get_bugbug_version()}")
         url = f"{formatted_base_url}/{db_type}.json.zst"
-        logger.info(f"Downloading {db_type} database from {url} to {path}.zst")
-        urlretrieve(url, f"{path}.zst")
-        assert os.path.exists(f"{path}.zst"), "Downloaded file exists"
-        logger.info(f"Decompressing {db_type} database")
-        self.decompress_file(path)
+        db.register(path, url, get_bugbug_version())
+        db.download(path, force=True, support_files_too=True)
```

Comment:
I think we don't need support_files here.

Reply:
Agreed